guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_prerequisites.adoc

```diff
@@ -17,11 +17,46 @@ Cloning is the recommended method because that prevents them being overwritten i
 To confirm if a template has been edited, you can view its *History* before you upgrade or view the changes in the audit log after an upgrade.
 In the {ProjectWebUI}, navigate to *Monitor* > *Audits* and search for the template to see a record of changes made.
 If you use the export method, restore your changes by comparing the exported template and the default template, manually applying your changes.
+* Pulp content migration occurs at {ProjectVersionPrevious}, before the upgrade to {ProjectVersion}.
+* Content migration can be run online, but uses processor, disk, and memory resources.
+Sync and content view publishing operations might take longer as a result.
+* If you have not pre-migrated Pulp content, the `PULP_CONTENT_PREMIGRATION_BATCH_SIZE` setting defines the number of content units processed at the same time.
+It affects the amount of RAM used and the I/O load.
+The lower the value, the longer `satellite-maintain content-prepare` takes to complete.
+The upgrade downtime is also longer if you have content left to migrate at that point.
+** The default value is 1000.
+** If the system has a hard disk drive, or there are concerns about I/O load, the recommended value is 50.
+** A lower value is not recommended.
+* Use the following method to set `PULP_CONTENT_PREMIGRATION_BATCH_SIZE`:
+. Create a directory and file:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ mkdir /etc/systemd/system/pulpcore-worker@.service.d/
+$ vi  /etc/systemd/system/pulpcore-worker@.service.d/settings.conf
+----
+with contents:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+[Service]
+User=pulp
+Environment=PULP_CONTENT_PREMIGRATION_BATCH_SIZE=1000
+----
+. Restart the `{foreman-maintain}` services using:
++
+[options="nowrap",  subs="verbatim,quotes,attributes"]
+----
+# systemctl daemon-reload
+# {foreman-maintain} service restart
+----
++
+For further information, see xref: migrating_pulp_content[Migrating Content to Pulp 3].
```

Comment:
we need to wrap this whole section in a ` ifdef::satellite[]`

Reply:
added